### PR TITLE
dapp: extend gitignore for server PR cache

### PR DESCRIPTION
**Pull request by weilbith**
The tool to serve the dApp for a specific PR does store some cache of
the CircleCI artifacts locally. These should be ignored by the SCM.

@andrevmatos it was too lazy to check there could be also other name schemes. Feel free to extend if there are more.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+serve_pr/[0-9]*/
```
Comment (kelsos) [inline .gitignore:2]: this is for running the script directly right?

Comment (weilbith) [reply .gitignore]: Yes, exactly.

**Comment by weilbith** [comment]
@andrevmatos thanks, good point. Accidentally missed that. Adopted for it.

**Comment by weilbith** [comment]
I'll rebase to merge the fixup commit. Then it is good to get merged.

**Pull request merged**